index.html

```diff
@@ -13,10 +13,10 @@
     <script type="text/javascript" class="remove">
       var respecConfig = {
         // specification status (e.g. WD, LCWD, NOTE, etc.). If in doubt use ED.
-        specStatus: "CG-FINAL",
+        specStatus: "FPWD",
 
         // the specification's short name, as in http://www.w3.org/TR/short-name/
-        shortName: "claims-data-model",
+        shortName: "verifiable-claims-data-model",
```

Comment:
We need to make sure this is vc-use-cases and vc-data-model - I thought those were the names we picked for short names?